src/UI/Panels/ClipboardPanel.cs

```diff
@@ -4,8 +4,9 @@ namespace UnityExplorer.UI.Panels
 {
     public class ClipboardPanel : UEPanel
     {
-        public static object Current { get; private set; }
-
+        public static List<object> ClipboardItems = new List<object>();
```

Comment:
I read the code and it is in FILO logic, so is it possible to refactor it using `System.Collection.Generic.Stack<object>`?
If it's possible to refactor it, it would eliminate the need for boundary checks, etc.

Reply:
I've never used a ```Stack``` before. I'll look into it.